### ISSUE TITLE
Some recipe tabs are no longer showing

### ISSUE THREAD

**Issue by MuteTiefling** [emilyploszaj/emi#683]
EMI version 1.1.12

Certain PNC recipe views are not showing in EMI only. Tested in a limited instance with fresh installs of PNC, JEI, and EMI to confirm.

https://github.com/TeamPneumatic/pnc-repressurized/issues/1344

**Comment by desht** [comment]
(PNC author here)

I see this logged in my client when recipes are processed:

```
java.lang.AbstractMethodError: Receiver class dev.emi.emi.jemi.impl.JemiRecipeSlotBuilder does not define or inherit an implementation of the resolved method 'abstract mezz.jei.api.gui.builder.IRecipeSlotBuilder addRichTooltipCallback(mezz.jei.api.gui.ingredient.IRecipeSlotRichTooltipCallback)' of interface mezz.jei.api.gui.builder.IRecipeSlotBuilder.
	at TRANSFORMER/pneumaticcraft@8.1.1/me.desht.pneumaticcraft.common.thirdparty.jei.JEIMemoryEssenceCategory.setRecipe(JEIMemoryEssenceCategory.java:62)
	at TRANSFORMER/pneumaticcraft@8.1.1/me.desht.pneumaticcraft.common.thirdparty.jei.JEIMemoryEssenceCategory.setRecipe(JEIMemoryEssenceCategory.java:49)
	at TRANSFORMER/emi@1.1.12+1.21+neoforge/dev.emi.emi.jemi.JemiRecipe.<init>(JemiRecipe.java:55)
	at TRANSFORMER/emi@1.1.12+1.21+neoforge/dev.emi.emi.jemi.JemiPlugin.register(JemiPlugin.java:263)
	at TRANSFORMER/emi@1.1.12+1.21+neoforge/dev.emi.emi.runtime.EmiReloadManager$ReloadWorker.run(EmiReloadManager.java:185)
```

Using JEI 19.14.0.138 and EMI 1.1.12.  Looks like it's related to the JEI tooltip callback API, which has changed in recent JEI builds (19.9+ I think?).

**Comment by emilyploszaj** [comment]
This is related to implementing new JEI APIs in 1.20.1 and 1.21, and has been resolved in EMI 1.1.13
